README.md

```diff
@@ -1,3 +1,4 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+To access the database, create a .env file for environment variables (.env.*deployment or test*) and set PGDATABASE to the desired database that you want to access(PGDATABASE=*insertDatabase*)
+
```

Comment:
You .env files should be .env.test and .env.development not .deployment. This matters because .deployment will require a URL too. The readMe should be specific on the database name instead of *insertDatabase* because the setup-sql has that database name in it.